config/sidekiq_scheduler.yml

```diff
@@ -271,3 +271,4 @@ TestUserDashboard::MirrorAccountAvailabilityLogsInBigQuery:
   cron: "0 */2 * * * America/New_York"
   class: TestUserDashboard::MirrorAccountAvailabilityLogsInBigQuery
   description: "Mirrors the test_user_dashboard_tud_account_availability_logs table in BigQuery."
+  
```

Comment:
Unnecessary change to this file.

Reply:
this change was committed by @thilton-oddball

Reply:
The newline at the end of the file had been removed and I put it back. This one is fine.